.sync.yml

```diff
@@ -3,36 +3,41 @@ Gemfile:
   optional:
     ":development":
       - gem: beaker
-        version: '~> 4.30'
+        version: '~> 5.0'
         from_env: BEAKER_VERSION
       - gem: beaker-abs
         from_env: BEAKER_ABS_VERSION
-        version: '~> 0.9'
-      - gem: beaker-pe
+        version: '~> 1.0'
       - gem: beaker-hostgenerator
         from_env: BEAKER_HOSTGENERATOR_VERSION
       - gem: beaker-rspec
         from_env: BEAKER_RSPEC_VERSION
+      # Prevent beaker-puppet from being installed on Ruby > 3.1 until
+      # beaker-puppet supports newer Rubies (PA-6136)
       - gem: beaker-puppet
         from_env: BEAKER_PUPPET_VERSION
-        version: '~> 1.22'
-      - gem: github_changelog_generator
-        version: '= 1.16.4'
-      - gem: concurrent-ruby
-        version: '= 1.1.10'
+        version: '~> 3.0'
+        condition: Gem::Requirement.create('< 3.2.0').satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))
       - gem: async
         version: '~> 1'
       - gem: beaker-module_install_helper
       - gem: beaker-puppet_install_helper
       - gem: nokogiri
+    ":system_tests":
+      - gem: voxpupuli-acceptance
 
 appveyor.yml:
   delete: true
 .travis.yml:
   delete: true
-.github/workflows/auto_release.yml:
-  unmanaged: false
 .github/workflows/release.yml:
   unmanaged: false
+.github/workflows/release_prep.yml:
+  unmanaged: false
```

Comment:
Nit, but I don't think we need to have any configuration for `release.yml` and `release_prep.yml` if all we're doing is saying that they're managed files. I tried removing these lines and running `pdk update` and nothing changed.